packages/ffe-core-react/src/typography/LinkText.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { bool, node, string } from 'prop-types';
+import classNames from 'classnames';
+
+const LinkText = (props) => {
+    const {
+        children,
+        className,
+        href,
+        underline,
+        ...rest
+    } = props;
+
+    return (
+        <a
+            className={classNames(
+                'ffe-link-text',
+                { 'ffe-link-text--no-underline': !underline },
+                className,
+            )}
+            href={href}
```

Comment:
Only thing I'm curious about is why this couldn't just be passed as part of `..rest`, it doesn't seem to be used for anything.

Reply:
True, but linting complained about an `a` without an `href` :( A tiny tiny bonus is that we get propType validation during development in case a link from a variable ever becomes null or undefined 😀